pkgs/by-name/tk/tkey-verification/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule rec {
```

Comment:
consider changing to the finalAttrs pattern.

Reply:
How would that be done in this case? I am quite new to making packages and Go is not usually my eco system of choice.

Reply:
Don't worry about it then :) it's just a new pattern some reviewers prefer but its not required yet